main/test/ca/uwaterloo/flix/library/TestMonoid.flix

```diff
@@ -17,29 +17,29 @@
 namespace TestMonoid {
 
     use Monoid.{empty, combine};
-    use Monoid.{combineAll};
+    use Monoid.{fold};
 
     /////////////////////////////////////////////////////////////////////////////
-    // combineAll                                                              //
+    // fold                                                                    //
```

Comment:
Do we not already have tests for fold?

Reply:
Yes, in `TestFoldable.flix`. Should I just delete them here?

Reply:
Yes, delete them.